sdk/tracing/azure-core-tracing-opencensus/README.md

```diff
@@ -0,0 +1,228 @@
+# Azure Tracing client library for Java
+This package enables distributed tracing across Azure SDK Java libraries through [Opencensus][opencensus]. OpenCensus is an open source, vendor-agnostic, single distribution of libraries to provide metrics collection and distributed tracing for services. 
+The Azure core tracing package provides:
+- Context propagation, in order to correlate activities and requests between services with an initial customer action.
+- Tracing user requests to the system, allowing to pinpoint failures and performance issues.
+
+[Source code][source_code] | [API reference documentation][api_documentation] | [Product
+documentation][api_documentation] | [Samples][samples]
+
+## Getting started
+### Prerequisites
+- Java Development Kit (JDK) with version 8 or above
+- [Maven][maven]
+```xml
+<dependency>
+    <groupId>io.opencensus</groupId>
+    <artifactId>opencensus-api</artifactId>
+    <version>0.21.0</version>
+</dependency>
+
+<dependency>
+    <groupId>io.opencensus</groupId>
+    <artifactId>opencensus-impl</artifactId>
+    <version>0.21.0</version>
+</dependency>
+```
+
+### Adding the package to your product
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-core-tracing-opencensus</artifactId>
+    <version>1.0.0-preview.3</version>
+</dependency>
+```
+
+### Default HTTP Client
+All client libraries support a pluggable HTTP transport layer. Users can specify an HTTP client specific for their needs by including the following dependency in the Maven pom.xml file:
+
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-core-http-netty</artifactId>
+    <version>1.0.0-preview.4</version>
+</dependency>
+```
+
+This will automatically configure all client libraries on the same classpath to make use of Netty for the HTTP client. Netty is the recommended HTTP client for most applications. OkHttp is recommended only when the application being built is deployed to Android devices.
+
+If, instead of Netty it is preferable to use OkHTTP, there is a HTTP client available for that too. Simply include the following dependency instead:
+
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-core-http-okhttp</artifactId>
+    <version>1.0.0-preview.4</version>
+</dependency>
+```
+
+### Configuring HTTP clients
+When an HTTP client is included on the classpath, as shown above, it is not necessary to specify it in the client library [builders](#create-eventhubs-builders), unless you want to customize the HTTP client in some fashion. If this is desired, the `httpClient` builder method is often available to achieve just this, by allowing users to provide a custom (or customized) `com.azure.core.http.HttpClient` instances.
```

Comment:
Does this `#create-eventhubs-builders` anchor work? I don't see any other heading wit hthat name.

Reply:
updated, to have a hyperlink to evenhubs heading.